### DISCUSSION TITLE
Vercel Deployement

### DISCUSSION THREAD

**Discussion by elsycharles** [community/community#133900] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is it possible to see how many people have clicked and consulted the deployement link in Vercel ?

**Comment by FlutterSmith** [comment, marked as answer]
Yes, it is possible to track how many people have clicked and consulted the deployment link in Vercel by using analytics tools. Here are a few methods you can use:

Vercel Analytics (Pro Plan) - Vercel offers built-in analytics for Pro plan users. This feature allows you to see page views, unique visitors, and other metrics directly from the Vercel dashboard.

Google Analytics - You can integrate Google Analytics into your Vercel deployment. By adding the Google Analytics tracking code to your project, you can track page views, user behavior, and other detailed metrics.

Third-Party Analytics Tools - Other tools like Plausible, Fathom, or Mixpanel can be integrated into your Vercel deployment. These tools often offer simple setup processes and can provide various analytics features.

Server Logs - If you have access to server logs, you can analyze these logs to see how many times your site has been accessed. This requires more manual effort and technical know-how.

**Reply by elsycharles** [reply]
Thank you so much, I've been documenting about Google Analytics and I used it and it works well. Thank you.
